draft-ietf-rats-corim.md

```diff
@@ -255,8 +256,20 @@ convention - e.g., {{Section 4.4.1.5 of -psa-token}}).
 ~~~
 
 # CoRIM
```

Comment:
The heading should spell out the name: "Concise Reference Integrity Manifest (CoRIM)"